scripts/compileDOIs.R

```diff
@@ -35,7 +35,7 @@ extractDOIs <- function(files){
 # Run the above function
 extractDOIs(Files)
 # Save the tibble output as a CSV
-write_csv(DOIs, file = here("data/AR6_DOIs.csv"))
+write_csv(DOIs, file = here("data/AR6_DOIs_test.csv"))
```

Comment:
Not sure why "_test" was added–given that doesn't match the output in the /data directory, I might advise rejecting the changes to line 38.

Reply:
Yea, good catch! I'll see if I can reject or remove.